### PR TITLE
adding usage script

### PR DESCRIPTION
**Pull request by mcough2**
Adding a script to generate usage data

### CODE REVIEW

File: scripts/add-plan-to-customer-with-price-adjustments/generate-usage-data/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Overview
+
+Python3 script which shows how to generate usage data for two customers.
```
Comment (eliakaris) [inline scripts/add-plan-to-customer-with-price-adjustments/generate-usage-data/README.md:3]: can you add this to its own folder, maybe:
scripts/usage/

then we can have "generate-usage" and a get-usage script in the future

Comment (mcough2) [reply scripts/add-plan-to-customer-with-price-adjustments/generate-usage-data/README.md]: done

File: scripts/generate-usage-data/generate-usage-data.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+from datetime import datetime, timedelta, timezone
+from pprint import pprint
+import random
+import uuid
+import json
+
+import requests
+
+token = '<API Token>'
```
Comment (eliakaris) [inline scripts/generate-usage-data/generate-usage-data.py:9]: are you using this? I dont see it?

File: scripts/generate-usage-data/generate-usage-data.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+for _ in range(58):
+    userId.append(uuid.uuid4().hex)
+
+def ingest(events):
+    remainder = events
+    with open('ingest.json', 'w') as f:
+        json.dump(events, f)
+    f.close()
+    print('events sent successfully')
```
Comment (eliakaris) [inline scripts/generate-usage-data/generate-usage-data.py:23]: these events arent really sent, they are stored in a file.  do you want to send them?

File: scripts/generate-usage-data/generate-usage-data.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+from pprint import pprint
+import random
+import uuid
+import json
+
+import requests
+
+token = '<API Token>'
+EnterpriseCustomer = 'customer_id'
+StandardCustomer = 'customer_id'
```
Comment (eliakaris) [inline scripts/generate-usage-data/generate-usage-data.py:11]: these should come in from the command line args, check out the argparser, it also supports a list of args, its super neat!

File: scripts/generate-usage-data/generate-usage-data.py [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    f.close()
+    print('events sent successfully')
+
+def random_monthly_target(n):
+    return random.randint(n, 2*n)
+
+events = []
+for cid in CustomerIds: 
+    customerId = cid
+    t = datetime.now(timezone.utc) - timedelta(days=33)
```
Comment (eliakaris) [inline scripts/generate-usage-data/generate-usage-data.py:31]: why remove 33 days?

Comment (eliakaris) [reply scripts/generate-usage-data/generate-usage-data.py:31]: make this configurable?

File: scripts/generate-usage-data/generate-usage-data.py [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+                "event_type": event_type,
+                "customer_id": customerId,
+                "properties": properties,
+            })
+        if cid == StandardCustomer:
+                append_event('gb_logs_ingested', {
+                    "gb_ingested":random_monthly_target(10000),
+                    "region": random.choice(["eu-west-1","us-east-1"]),
+                    "cloud": random.choice(["aws","gcp","azure"])
+                })
```
Comment (eliakaris) [inline scripts/generate-usage-data/generate-usage-data.py:46]: i'd simplify this and just ignore the "customer type"

File: scripts/generate-usage-data/generate-usage-data.py [hunk 7]
```diff
@@ -17,10 +17,20 @@
 
 def ingest(events):
     remainder = events
-    with open('ingest.json', 'w') as f:
-        json.dump(events, f)
-    f.close()
-    print('events sent successfully')
+    while len(remainder) > 0:
+        pprint(remainder[:100])
```
Comment (eliakaris) [inline scripts/generate-usage-data/generate-usage-data.py:21]: dont print this :)

File: scripts/generate-usage-data/generate-usage-data.py (hunk 5 above)
Comment (mcough2) [reply scripts/generate-usage-data/generate-usage-data.py:41]: You are only allowed to backfill data 34 days in the past...this basically send is backfilled usage data

File: scripts/generate-usage-data/generate-usage-data.py (hunk 4 above)
Comment (mcough2) [reply scripts/generate-usage-data/generate-usage-data.py:11]: I prefer it this way, but would be open to understand what you think command line args are better

**Pull request closed without merging**